### PR TITLE
Add MutatingWebhookConfiguration and ValidatingWebhookConfiguration t…

### PR DESCRIPTION
**Pull request by kishoregv**
…o default Map.yaml

### CODE REVIEW

**Comment by kishoregv** [comment]
Ping @hickeyma Can you please review this PR ?

File: config/Map.yaml [hunk 1]
```diff
@@ -119,3 +119,11 @@ mappings:
...
     deprecatedInVersion: "v1.17"
     removedInVersion: "v1.22"
+  - deprecatedAPI: "apiVersion: admissionregistration.k8s.io/v1beta1\nkind: MutatingWebhookConfiguration"
+    newAPI: "apiVersion: admissionregistration.k8s.io/v1\nkind: MutatingWebhookConfiguration"
+    deprecatedInVersion: "v1.16"
+    removedInVersion: "v1.19"
+  - deprecatedAPI: "apiVersion: admissionregistration.k8s.io/v1beta1\nkind: ValidatingWebhookConfiguration"
+    newAPI: "apiVersion: admissionregistration.k8s.io/v1\nkind: ValidatingWebhookConfiguration"
+    deprecatedInVersion: "v1.16"
+    removedInVersion: "v1.19"
```
Comment (hickeyma) [inline config/Map.yaml:129]: As these APIs are not part of the [Helm install order list](https://github.com/helm/helm/blob/master/pkg/releaseutil/kind_sorter.go#L31-L66) then I am not sure they should be added by default to the mapping list. They can always be added on the fly.

Comment (Gearheads) [reply config/Map.yaml:129]: hi @hickeyma, how do you add them on the fly?

do we just need to update the `Map.yaml`?

Just wondering if you had any other recommendations.

Comment (hickeyma) [reply config/Map.yaml:129]: @Gearheads Sure, you can just add it to the file on the fly. Check out [API Mapping](https://github.com/hickeyma/helm-mapkubeapis#api-mapping) for more details.

Comment (Gearheads) [reply config/Map.yaml:129]: ok, understood.  thank you for the reply @hickeyma.

**Comment by hickeyma** [comment]
Closing as APIs are not part of the [Helm install order list](https://github.com/helm/helm/blob/master/pkg/releaseutil/kind_sorter.go#L31-L66) and APIs can be added if needed using the [API Mapping](https://github.com/hickeyma/helm-mapkubeapis#api-mapping) file.

**Pull request closed without merging**